lib/storage/metadata/file/server.js

```diff
@@ -50,12 +55,52 @@ class MetadataFileServer {
         this.logger = new Logger('MetadataFileServer', options);
     }
 
+    genUUIDIfNotExists() {
+        const uuidFile = `${this.metadataPath}/uuid`;
+
+        try {
+            fs.accessSync(uuidFile, fs.F_OK | fs.R_OK);
+        } catch (e) {
+            if (e.code === 'ENOENT') {
+                const v = uuid.v4();
+                const fd = fs.openSync(uuidFile, 'w');
+                fs.writeSync(fd, v.toString());
+                fs.closeSync(fd);
+            } else {
+                throw e;
+            }
+        }
+    }
+
+    printUUID() {
+        const uuidFile = `${this.metadataPath}/uuid`;
+        const uuidValue = fs.readFileSync(uuidFile);
+        this.logger.info(`This deployment's identifier is ${uuidValue}`);
+    }
+
     /**
      * Start the metadata server and listen to incoming connections
      *
      * @return {undefined}
      */
     startServer() {
+        fs.accessSync(this.metadataPath, fs.F_OK | fs.R_OK | fs.W_OK);
+
+        const warning =
+                  'WARNING: Synchronization directory updates are not ' +
```

Comment:
I'm not convinced this will make it more readable: AFAIK, all spaces present in a template string in the code file are part of the string, so I would have to align the second line to the left margin, which makes indentation confusing. Otherwise I can concatenate two template strings, but then it's not much different than using plain strings.

Please correct me if I'm wrong.